.env.example

```
@@ -0,0 +1,48 @@
+#   cp .env.example .env && ./scripts/stack-up.sh
+#
+# This stack is a LOCAL TESTING TOOL. It binds to 127.0.0.1 only, and it is not a
+# deployment artifact — see docs/specs/SPEC-004-env-config-and-container-stack.md.
+
+# ── Database ──────────────────────────────────────────────────────────────────
+# The stack's own database, separate from the one compose.yaml starts for native
+# development. Local only, so the values are not secret — but POSTGRES_PASSWORD has no
+# default on purpose: an empty password would make the stack fail late and confusingly.
+POSTGRES_DB=storeit
+POSTGRES_USER=storeit
+POSTGRES_PASSWORD=storeit
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The POSTGRES_PASSWORD key should go before the POSTGRES_USER key

(UnorderedKey)